chart/INSTALL

```diff
@@ -12,3 +12,28 @@ kubectl create namespace airflow
 
 ## Install the chart in that namespace
 helm install airflow -n airflow .
+
+# Naming conventions
```

Comment:
Can you move this section to the docs instead? Maybe on the homepage?

Reply:
awesome - moved to docs/helm-chart/index.rst in the next commit